code/style.css

```diff
@@ -5,3 +5,212 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+
```

Comment:
Hmm... There are some wierd indention going on in this file. It needs some love 😀